deploy/README.md

```diff
@@ -221,7 +221,7 @@ extenders:
   nodeCacheCapable: false
   weight: 100 ## EDIT THIS FIELD ##
   managedResources:
-  - name: "topolvm.cybozu.com/capacity"
```

Comment:
Would please add a note to use the `topolvm.cybozu.com` symbol instead of `topolvm.io` when updating old topolvm clusters?